src/source/view/controls/Button.bs

```diff
@@ -9,7 +9,7 @@ namespace mv.controls
     '+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
     @alwaysnotify
-    public clicked as boolean
+    public selected as boolean
```

Comment:
please be sure to review my changes in this file. particularly calculateSize and layout funcions - I was happy with how I split some of that up.